proj/src/reducers/projects/projectsReducer.js

```diff
@@ -0,0 +1,49 @@
+import { handleActions, combineActions } from 'redux-actions';
+import { set, unset, flow } from 'lodash/fp'
+
+import { ADD_PROJECT, EDIT_PROJECT } from 'actions/types'
+const initialState = {
+    isLoading: false,
+    displayEditModal: false,
+    displayMode: 'list',
+    currProject: null,
+    projects: {
+        1: {id: 1, name: 'Project 1'},
+        2: {id: 2, name: 'Project 2'},
+        3: {id: 3, name: 'Project 3'},
+    }
+};
+
+
+export default handleActions({
+    [combineActions(ADD_PROJECT, EDIT_PROJECT,)]: (state, payload) =>
```

Comment:
would be better to put combined actions vertically so it's more readable
```suggestion
    [combineActions(
        ADD_PROJECT, 
        EDIT_PROJECT
    )]: (state, payload) =>
```